integr8ly/integreatly-operator
# MGDAPI-5754 - keycloak: v7.6.3-1

# Issue link
[MGDAPI-5754](https://issues.redhat.com/browse/MGDAPI-5754)

# What
Update keycloak to v7.6.3-1

# Verification steps

* Checkout master
* Install RHOAM
```
LOCAL=false make cluster/prepare/local                                                                               
LOCAL=false make code/run
```
* Allow for installation to complete
* Deploy webload-web app- [integr8ly/workload-web-app#rhoam-clusters](https://github.com/integr8ly/workload-web-app#rhoam-clusters)
* Stop running the operator
* Checkout this branch
* Re-run the operator
```
LOCAL=false make code/rerun
```
* Verify keycloak operator upgrades thats places and installation completes
```
 watch "echo RHSSO; oc get csv -n redhat-rhoam-rhsso-operator; oc get pods -n redhat-rhoam-rhsso; echo "";echo "USER SSO"; oc get csv -n redhat-rhoam-user-sso-operator; oc get pods -n redhat-rhoam-user-sso; echo ""; echo "RHMI CR"; oc get rhmi rhoam -n redhat-rhoam-operator -o jsonpath='{.status.stage}'"
```
* Visit the workload web app grafana dashboard and confirm any downtime measures
![image](https://github.com/integr8ly/integreatly-operator/assets/6575004/4f5213aa-6fe2-44d0-8dbb-1700055dd607)
